index.js

```diff
@@ -48,6 +48,11 @@ Wemo.prototype.load = function(setupUrl, cb) {
           cb.call(self, device);
         }
       }
+    } else {
+      debug('Error occurred connecting to device at %s', location);
+      if (cb) {
+        cb.call(self, null, err);
```

Comment:
How do you think about making the `err` go first? While that would break backwards compatibility it stays consistent with the other callbacks that all follow the "err first" pattern.